conda/base/context.py

```diff
@@ -592,7 +592,6 @@ def list_parameters(self):
             'bld_path',
             'conda_build',
             'croot',
-            'allow_conda_downgrades',
```

Comment:
why is this removed?  I think it has to stay here?

Reply:
This is the list of 'unlisted parameters'

Reply:
Of course... Sorry.